DEVELOPMENT.md

```diff
@@ -0,0 +1,30 @@
+# Development
+
+The simples way to test the code is to run it from local.
+If you have `capoci-controller-manager ` running in your management cluster, please scale down the deployment
+```bash
+kubectl scale deployment/capoci-controller-manager  --replicas=0 -n cluster-api-provider-oci-system
+```
+Then run the following commands.
+```bash
+export AUTH_CONFIG_DIR="<repo-path>/auth-config.yaml"
+make run
+```
+This will run the code against the local management cluster.
+
+Please execute the following steps if you want to create an image from the repository
```

Comment:
I'm not sure I understand what's going on here. Do you mean if "you want to use a container image your repository"?

You also need to let the developer know what this image is for and which namespace this is referring to (K8s or OCI namespace).

Reply:
- [ ] Review wording
- [ ] Add steps to find namespace